packs/feats/touch-of-lunacy.json

```diff
@@ -28,7 +28,74 @@
             "remaster": true,
             "title": "Pathfinder Howl of the Wild"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
```

Comment:
We may want to tag the strikes instead using ItemAlteration then predicate on the tag.

Reply:
Is there an existing naming convention for this type of tag?

Reply:
user `werecreature-unarmed-strike` as the tag. Matches what we have been using so far